tests/integration/test.replication.js

```diff
@@ -839,7 +900,9 @@ adapters.forEach(function (adapters) {
             return db2.changes({style: 'all_docs'});
           }).then(function (res2) {
             changes2 = simplifyChanges(res2);
-
+            if (testUtils.isSyncGateway() && adapters[0] === 'http') {
+              return true;
+            }
```

Comment:
Surprised that this one fails. this was a really tricky bug that we managed to fix recently (https://github.com/pouchdb/pouchdb/pull/3165 #3136). Maybe should check to make sure this isn't a bug in CSG?